mllib/src/main/scala/org/apache/spark/ml/classification/DecisionTreeClassifier.scala

```diff
@@ -81,7 +81,7 @@ class DecisionTreeClassifier @Since("1.4.0") (
    * E.g. 10 means that the cache will get checkpointed every 10 iterations.
    * This is only used if cacheNodeIds is true and if the checkpoint directory is set in
    * [[org.apache.spark.SparkContext]].
-   * Must be >= 1.
+   * Must be {@literal >= 1}.
```

Comment:
Actually, it might be already fine and too trivial to mention but in scaladoc, inlined tag becomes monospace text and this will be `<code>&gt;= 1</code>` in HTML whereas it becomes `&gt;= 1` in javadoc.

So, I just decided to use `{@literal >}` instead of `{@literal >= 1}` to minimise the appearance differences bettwen javadoc and scaladoc. This is trivial and maybe not meant to be forced but I just wanted note.